### ISSUE TITLE
Skip link scrolls too far down the page

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#5273]
**Describe the bug**

Navigating to the main content ID (#main-content) using the skip-link takes me to the 'Stake your ETH' section (well lower in the page than I should be taken to). This bug is persistent across all pages I tested.

**Other pages tested:**
- DeFi 
- governance
- Developer docs, 
- Developer tutorials

**To Reproduce**

1. Go to 'https://ethereum.org/en/run-a-node/'
2. Hit tab to see skip link
3. Hit enter to visit main content

**or**

Visit https://ethereum.org/en/run-a-node/#main-content

**Expected behavior**

A skip link should take the user to the first meaningful content after the navigation bar.

**Desktop (please complete the following information):**

- OS: macOS Monteray 
- Browser: Brave Browser
- Version: 1.34.80

**Additional context**

The main-content ID is on the correct element.

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by vluna** [comment]
@minimalsm I am not 100% sure about the solution for this, but after looking for a couple of days online I think I found a solution.

**Comment by vluna** [comment]
@minimalsm I have created the PR, I am sure maybe this is not what is expected but thought I will give it a try.
